MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -90,16 +196,60 @@
 #pragma mark -
 #pragma mark acquireToken using User
 
+/*!
+    Acquire a token requiring interaction for a previously used user
+ 
+    @param  scopes          Scopes to request from the server, the scopes that come back
+                            can differ from the ones in the original call
+    @param  user            A user object retrieved from the application object that the
+                            interactive authentication flow will be locked down to.
+    @param  completionBlock The completion block that will be called when the authentication
+                            flow completes, or encounters an error.
+ */
 - (void)acquireTokenForScopes:(NSArray<NSString *> *)scopes
                          user:(MSALUser *)user
               completionBlock:(MSALCompletionBlock)completionBlock;
 
+/*!
+    Acquire a token requiring interaction for a previously used user
```

Comment:
Would `specified` be better? I wanted to make explicit that you have to have acquired a token for the user with the library prior to using this method, to differentiate it from the versions of AT that don't take a user.